base/src/lib.rs

```diff
@@ -584,49 +584,49 @@ pub type Tpm2bOperand = Tpm2bDigest;
 #[repr(C)]
 #[derive(Clone, Copy, PartialEq, Debug)]
 pub struct Tpm2bData {
-    size: u16,
+    pub size: u16,
```

Comment:
I do not think tpm2b family of types should expose size and buffer content. It might be better to expose that functionality over 3 API.

```
impl AsRef<[u8]> for Tpm2b*

impl AsMut<[u8]> for Tpm2b*
```